process_scenario_weo_2024_hybrid_in_ev.R

```diff
@@ -0,0 +1,132 @@
+logger::log_info("WEO 2024: Setting WEO 2024 config.")
+
+weo_2024_raw_path <- config[["weo_2024_raw_path"]]
+weo_2024_ext_data_regions_raw_filename <- config[["weo_2024_ext_data_regions_raw_filename"]]
+weo_2024_ext_data_world_raw_filename <- config[["weo_2024_ext_data_world_raw_filename"]]
+iea_global_ev_2024_raw_filename <- config[["iea_global_ev_raw_filename"]]
+weo_2024_fig_chptr_3_raw_full_filename <- config[["weo_2024_fig_chptr_3_raw_full_filename"]]
+mpp_ats_raw_path <- config[["mpp_ats_raw_path"]]
+mpp_ats_raw_filename <- config[["mpp_ats_raw_filename"]]
+
+logger::log_info("WEO 2024: Setting WEO 2024 paths.")
+
+weo_2024_raw_full_path <-
+  file.path(
+    scenario_preparation_inputs_path,
+    weo_2024_raw_path
+  )
+
+weo_2024_ext_data_regions_raw_full_filepath <-
+  file.path(
+    weo_2024_raw_full_path,
+    weo_2024_ext_data_regions_raw_filename
+  )
+
+weo_2024_ext_data_world_raw_full_filepath <-
+  file.path(
+    weo_2024_raw_full_path,
+    weo_2024_ext_data_world_raw_filename
+  )
+
+iea_global_ev_2024_raw_full_filepath <-
+  file.path(
+    weo_2024_raw_full_path,
+    iea_global_ev_2024_raw_filename
+  )
+
+weo_2024_fig_chptr_3_raw_full_filepath <-
+  file.path(
+    weo_2024_raw_full_path,
+    weo_2024_fig_chptr_3_raw_full_filename
+  )
+
+mpp_ats_raw_full_path <-
+  file.path(
+    scenario_preparation_inputs_path,
+    mpp_ats_raw_path
+  )
+
+mpp_ats_raw_full_filepath <-
+  file.path(
+    mpp_ats_raw_full_path,
+    mpp_ats_raw_filename
+  )
+
+logger::log_info("WEO 2024: Checking that WEO 2024 filepaths exist.")
+
+stopifnot(fs::file_exists(weo_2024_ext_data_regions_raw_full_filepath))
+stopifnot(fs::file_exists(weo_2024_ext_data_world_raw_full_filepath))
+stopifnot(fs::file_exists(weo_2024_fig_chptr_3_raw_full_filepath))
+stopifnot(fs::file_exists(iea_global_ev_2024_raw_full_filepath))
+stopifnot(fs::file_exists(mpp_ats_raw_full_filepath))
+
+logger::log_info("WEO 2024: Loading WEO 2024 raw data.")
+
+weo_2024_ext_data_regions_raw <-
+  readr::read_csv(
+    file = weo_2024_ext_data_regions_raw_full_filepath,
+    show_col_types = FALSE
+  )
+
+weo_2024_ext_data_world_raw <-
+  readr::read_csv(
+    file = weo_2024_ext_data_world_raw_full_filepath,
+    show_col_types = FALSE
+  )
+
+weo_2024_fig_chptr_3_raw <-
+  tidyxl::xlsx_cells(
+    path = weo_2024_fig_chptr_3_raw_full_filepath
+  )
+
+iea_global_ev_2024_raw <-
+  readxl::read_xlsx(
+    path = iea_global_ev_2024_raw_full_filepath,
+    sheet = "electric-vehicle-sales-by-regio"
+  )
+
+iea_sales_share_ev <-
+  readxl::read_xlsx(
+    path = iea_global_ev_2024_raw_full_filepath,
+    sheet = "electric vehicle share-ev"
+  )
+
+iea_sales_share_bev_phev <-
+  readxl::read_xlsx(
+    path = iea_global_ev_2024_raw_full_filepath,
+    sheet = "electric-vehicle-share-bev-phev"
+  )
```

Comment:
this is not being used anywhere? can it be deleted?

Reply:
just removed! but then it needs a new review @cjyetman - thanks and sorry